### ISSUE TITLE
Examples on pixi website don't seem to be accessible?

### ISSUE THREAD

**Issue by Zireael07** [pixijs/pixi.js#6955]
I read on the main page that all pixi content is accessible. From a brief look through the accessibility code here on GitHub, it seems content should become accessible (i.e. generate matching HTML) on pressing TAB. However, when I press tab on the examples page https://pixijs.io/examples/#/demos-basic/container.js, nothing happens.

**Comment by bigtimebuddy** [comment]
PixiJS has an accessibility module which adds HTML elements for interactive display objects. Because the example you point out has no interactive elements, it would not have any tab-able elements.

Examples that _do_ have interactivity should have accessibility enabled. Would recommend filing an issue or adding a PR to that repo: https://github.com/pixijs/examples.

**Comment by Zireael07** [comment]
Main page should be clarified then, that it only applies to interactive elements.
